idx/nid.go

```
@@ -53,56 +55,57 @@ func refNIDsTo[T Type](ids []*nid) []*ID[T] {
-func fromNID(id string) (nid, error) {
+func fromNID(id string) (*nid, error) {
 	parsedID, e := parseID(id)
 	if e != nil {
-		return nid{}, ErrInvalidID
+		return nil, ErrInvalidID
 	}
-	return nid{id: parsedID}, nil
+	return &nid{id: parsedID}, nil
 }
 
 func refNIDTo[T Type](n *nid) *ID[T] {
```

golangci-lint (1.62.2)

func `refNIDTo` is unused

(unused)